services/blockchain-indexer/shared/dataService/business/dpos/dposVotesSent.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const BluebirdPromise = require('bluebird');
+
+const {
+	getBase32AddressFromHex,
+	getHexAddressFromBase32,
+	getIndexedAccountInfo,
+} = require('../../../utils/accountUtils');
+const { getAddressByName } = require('../../../utils/delegateUtils');
+const { parseToJSONCompatObj } = require('../../../utils/parser');
+const { requestConnector } = require('../../../utils/request');
+
+const normalizeVote = vote => {
+	const normalizedVote = parseToJSONCompatObj(vote);
+	normalizedVote.delegateAddress = getBase32AddressFromHex(vote.delegateAddress);
```

Comment:
Don't we receive the address in the base32 format?

Reply:
No, we receive binary address from the node.